doc-content/enterprise-only/openshift/customimage-build-step.adoc

```diff
@@ -0,0 +1,14 @@
+. Build the custom image using the `Dockerfile`. Supply the fully qualified name for the image, including the registry name. You must use the same version tag as the version of the base image. Enter the following command:
```

Comment:
Do you enter the command that follows to build? If so, I would say:
To build the ... enter the following command. Replace registry_adrress and image_name with....